docs/administration.landing-zones.md

```diff
@@ -16,3 +16,7 @@ Please consult the [operator documentation](./meshstack.index.md) of the respect
 
 Disabled Landing Zones can't be assigned to projects anymore. If the Landing Zone has already been assigned to a project,
 this assignment will remain, but customer admins can no longer create new projects using this Landing Zone.
+
+## Configure Landing Zone tags
```

Comment:
d: Just a question but restricted tags do not work here right? Should we mention here that the use of `restricted` is not possible?